app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,27 @@
+<% @comments.each do |c| %>
```

Comment:
normally partial `_comment` have to render only one Comment record, so that this iterator would be good to replace by `render @comments` in views that renter this partial